package.json

```diff
@@ -23,7 +23,7 @@
     "lint:fix": "tsc --noEmit && eslint \"**/*.{js,ts,tsx}\" --quiet --fix",
     "copy-static-assets": "ts-node scripts/copyStaticAssets.ts",
     "watch-debug": "yarn copy-static-assets && concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"cyan.bold,green.bold\" \"yarn watch-ts\" \"yarn serve-debug\"",
-    "watch-node": "nodemon dist/server/main.js  --signal SIGTERM",
+    "watch-node": "nodemon --config nodemon.json",
```

Comment:
Yes, the nodemon.json file should be committed to the repository. This file contains the configuration settings for nodemon, which are essential for the development workflow. By committing this file, we ensure that all developers working on the project have a consistent setup and don't need to manually configure nodemon on their local machines. This helps to prevent configuration drift and makes it easier for new team members to set up their development environment.

Reply:
So, this file is missing in the PR. Please add it so it can be reviewed.

Reply:
Based on our discussion, we identified that the real issue was the dist/**/* path being ignored in the nodemonConfig. To resolve this, we reverted the script to its original state and removed the dist/**/* path from the ignored files in nodemonConfig.